### ISSUE TITLE
TypeError on forceRender() 

### ISSUE THREAD

**Issue by yababay** [grid-js/gridjs#83]
Hi folks, thank you for your great work! I use gridjs instead of bootstrap table and yours is sure better, lighter and more flexible.

But there is a bug. I get "TypeError: this.config is undefined" at grid.ts:81:9 when I use  forceRender() method.

I have a function, that dynamically updates my grid with server side data:

```javascript
// blah-blah...
fetch(url, {body, method, headers})
        .then(res=> res.status == 200 && res.json() || Promise.reject(res.status))
        .then(obj=> {
            window.location.hash = `#/${relation}`
            window.jQuery(`#${id}`).modal('hide')
            grid.forceRender()
        })
        //.catch(err => alert(err))
```
Something strange happens here: when I call fetch with POST method (adding a row), grid.forceRender() works fine as many times as I call it. But when I use PUT or DELETE methods - the error occurs. There is no network errors. Only "TypeError: this.config is undefined".

I get the error on Firefox browser, Linux OS. Before writing this message I have made `yarn upgrade gridjs`, but the bug is still here.

Could you explain or fix it?

**Comment by afshinm** [comment]
hey @yababay 

Have you tried using the `server` attributes instead of calling `fetch` yourself? see https://gridjs.io/docs/server-side

**Comment by afshinm** [comment]
oh and it's likely that you're calling forceRender() before render(): https://github.com/grid-js/gridjs/blob/master/src/grid.ts#L82

I will fix that line though.

**Comment by afshinm** [comment]
Fixed! Install 1.6.2

also you don't have to call forceRender() unless you're changing an existing config object or an existing Grid.js instance. Happy to investigate more if you create a jsFiddle/Codesandbox.

**Comment by yababay** [comment]
Thank you, for detailed replies, Afshin. 

I have upgraded to 1.6.2 now, but the bug appeared again. It seems that  the problem is really on my side. I will continue to analyze and try to overcome it, and will inform you about results here.

**Comment by yababay** [comment]
![Без имени](https://user-images.githubusercontent.com/60526241/84988275-66495680-b14a-11ea-9b11-2c107a2baeaa.png)
Here is a screenshot. Yes, the new exception appeared, but, as you can see, the grid exists and is attached to a DOM element (the `render()` was called once). Moreover, when I added last row in the table some seconds before I did the screenshot, the `forceRender()` have called correctly.

**Comment by yababay** [comment]
Hi all, I have found and fixed error in my code. I have one instance of greed common for 3 react components, let's call them Table, AddEditor and UpdateEditor. 
```
<div>
    <AddEditor grid={grid}/>
    <UpdateEditor grid={grid}/>
    <Table grid={grid}/>
</div>
```
Only Table calls the `render()` method, but editors are called **always after** Table is initiated. It seems that editors should get initiated (by Table) instance of the greed, but I had the error described above in this code. 

Finally I guessed that the order of components matter, and I have changed it this way: 
```
<div>
    <Table grid={grid}/>
    <AddEditor grid={grid}/>
    <UpdateEditor grid={grid}/>
</div>
```
Now the `forceUpdate` method works correctly.
